include/zephyr/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -187,6 +187,7 @@ SECTIONS
 	__rodata_region_start = .;
 
 #include <zephyr/linker/common-rom.ld>
+#include <snippets-rom-sections.ld>
```

Comment:
Like for the other occcurences of snippet, this should be documented with something of the form
```
/* Located in generated directory. This file is populated by calling
 * zephyr_linker_sources(ROM_SECTIONS ...). <description of typical use>.
 */
#include <snippets-rom-sections.ld>
```